rock-solid/wordpress-mobile-pack
# Update Radio Button And Checkbox CSS

## Ticket
https://trello.com/c/WNLYW87q/49-check-radiobuttons-checkboxes-on-the-wpmp-wpmp-pro-plugin

## Changes Made
* Update Radio Button And Checkbox CSS


## Steps to Test
1. Checkout this branch
2. Check the radio buttons / checkboxes on admin area
